packages/ui/src/components/notifications/store.ts

```diff
@@ -0,0 +1,22 @@
+import { writable } from 'svelte/store'
+import { getContext } from 'svelte'
+
+import { Notification } from './Notification'
+import { addNotification, removeNotification } from './actions'
+
+const store = writable<Notification[]>([])
+
+interface NotificationsContext {
+  subscribe: typeof store.subscribe
+  addNotification: (notification: Notification) => void
+  removeNotification: (notificationId: string) => void
+}
+
+export const notificationsContextKey = 'notifications-context'
+export const getNotificationsContext = (): NotificationsContext => getContext(notificationsContextKey)
```

Comment:
I do not think we need setContext/getContext here. You could just import store and use it as $store in Svelte.

Reply:
Done.